src/sys/unix.rs

```diff
@@ -440,16 +429,18 @@ impl SockAddr {
     }
 }
 
+#[cfg(all(feature = "all", any(target_os = "android", target_os = "linux")))]
```

Comment:
Same as above, this `cfg` is not needed (the `cfg_attr` is).

Reply:
I did a similar thing to what I did above; factor out the `#[cfg]` to the `impl` block instead of each item in the `impl` block individually. The effect of the `#[cfg]` is actually the exact same, it just avoids duplicating the attributes.